rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -1015,4 +1062,31 @@ public boolean changeStartAt(JobId jobId, String startAt)
         }
         return isJobStartAtChanged;
     }
+
+    @Override
+    public TaskFlowJob getInitialJobContent(JobId jobId)
```

Comment:
no, not add extra info on the job, but duplicate the job by adding or replacing some general info